lib/cstring.h

```diff
@@ -366,6 +368,9 @@ inline std::string &operator+=(std::string &a, cstring b) {
     return a;
 }
 
+inline cstring cstring::newline = cstring::literal("\n");
```

Comment:
Shall we combine declaration and definition and make them `static const inline`?

Reply:
As discussed offline, it seems that cstring does not have a destructor and the compiler is not happy making cstring a const value as shown here 
`error: calling 'literal<const char, 2UL, void>' with incomplete return type 'cstring'
  274 |     static const inline cstring newline = literal("\n");`
Marking as resolved.